bundles/org.openhab.binding.omnikinverter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -9,20 +9,93 @@
 		<description>Thing for OmnikInverter Binding</description>
 
 		<channels>
-			<channel id="power" typeId="power"/>
-			<channel id="powerAC1" typeId="powerAC1"/>
-			<channel id="powerAC2" typeId="powerAC2"/>
-			<channel id="powerAC3" typeId="powerAC3"/>
-			<channel id="currentPV1" typeId="currentPV1"/>
-			<channel id="currentPV2" typeId="currentPV2"/>
-			<channel id="currentPV3" typeId="currentPV3"/>
-			<channel id="voltagePV1" typeId="voltagePV1"/>
-			<channel id="voltagePV2" typeId="voltagePV2"/>
-			<channel id="voltagePV3" typeId="voltagePV3"/>
-			<channel id="energyToday" typeId="energyToday"/>
-			<channel id="energyTotal" typeId="energyTotal"/>
+			<channel id="power" typeId="system.electric-power">
+				<label>Instantaneous Power</label>
+				<description>The instantaneous power generation for output 1</description>
+			</channel>
+			<channel id="powerAC1" typeId="system.electric-power">
+				<label>Instantaneous Power AC1</label>
+				<description>The instantaneous power generation for output 1</description>
+			</channel>
+			<channel id="powerAC2" typeId="system.electric-power">
+				<label>Instantaneous Power AC2</label>
+				<description>The instantaneous power generation for output 2</description>
+			</channel>
+			<channel id="powerAC3" typeId="system.electric-power">
+				<label>Instantaneous Power AC3</label>
+				<description>The instantaneous power generation for output 3</description>
+			</channel>
+			<channel id="voltageAC1" typeId="system.electric-voltage" advanced="true">
```

Comment:
Does this work? IIRC "advanced" has to be configured on channel type level, so I would assume an exception is being thrown when the binding is started. See the XSD: https://www.openhab.org/schemas/thing-description-1.0.0.xsd

Another question is: Why advanced? The would be a change from previous definition.

Reply:
Reverted in [f25b9b2](https://github.com/openhab/openhab-addons/pull/14244/commits/f25b9b2a30f151bf75573d4c7b079c936a1dfbf0)